bullet_train-themes/app/views/themes/base/attributes/_belongs_to.html.erb

```diff
@@ -3,12 +3,12 @@
 <% disable_link ||= false %>
 <% url ||= nil %>
 
-<% if object.send(attribute).present? %>
+<% if object.respond_to?(attribute) %>
```

Comment:
Same here with `object.public_send(attribute).present?`. Fixing these two spots should make the tests pass.